test/unit/YearnCurveVaultOperator.unit.ts

```diff
@@ -0,0 +1,770 @@
+
+            // User1 creates the portfolio/NFT
+            await expect(
+                context.nestedFactory
+                    .connect(context.user1)
+                    .create(0, [{ inputToken: ETH, amount: ethToDepositAndFees, orders, fromReserve: false }], {
+                        value: ethToDepositAndFees,
+                    }),
+            ).to.be.revertedWith("NF: OPERATOR_CALL_FAILED");
+        });
+
+        it("Create with ETH and Deposit (uint256) in yearn 3crypto with WETH", async () => {
+            // All the amounts for this test
+            const ethToDeposit = appendDecimals(1);
+            const ethToDepositAndFees = ethToDeposit.add(getExpectedFees(ethToDeposit));
+
+            const ethBalanceBefore = await context.user1.getBalance();
+
+            // Orders to Deposit in yearn
+            let orders: utils.OrderStruct[] = utils.getYearnCurveDepositOrder(context, context.yearnVaultAddresses.triCryptoVault, context.WETH.address, ethToDeposit);
+
+            // User1 creates the portfolio/NFT
+            const tx = await context.nestedFactory
+                .connect(context.user1)
+                .create(0, [{ inputToken: ETH, amount: ethToDepositAndFees, orders, fromReserve: false }], {
+                    value: ethToDepositAndFees,
+                });
+
+            // Get the transaction fees
+            const txFees = await tx.wait().then(value => value.gasUsed.mul(value.effectiveGasPrice));
+
+            // User1 must be the owner of NFT n°1
+            expect(await context.nestedAsset.ownerOf(1)).to.be.equal(context.user1.address);
+
+            const mockERC20Factory = await ethers.getContractFactory("MockERC20");
+            const vault = mockERC20Factory.attach(context.yearnVaultAddresses.triCryptoVault);
+
+            //  Tokens in vault
+            const vaultBalanceReserve = await vault.balanceOf(context.nestedReserve.address);
+            expect(vaultBalanceReserve).to.not.be.equal(BIG_NUMBER_ZERO);
+
+            expect(await context.user1.getBalance()).to.be.equal(ethBalanceBefore.sub(ethToDepositAndFees).sub(txFees));
+
+            // The FeeSplitter must receive the right fee amount
+            expect(await context.WETH.balanceOf(context.feeSplitter.address)).to.be.equal(
+                getExpectedFees(ethToDeposit),
+            );
+
+            const expectedNfts = [
+                {
+                    id: BigNumber.from(1),
+                    assets: [{ token: context.yearnVaultAddresses.triCryptoVault, qty: vaultBalanceReserve }],
+                },
+            ];
+
+            const nfts = await context.nestedAssetBatcher.getNfts(context.user1.address);
+
+            expect(JSON.stringify(utils.cleanResult(nfts))).to.equal(JSON.stringify(utils.cleanResult(expectedNfts)));
+        });
+
+        it("Create with ETH and Deposit (int128) in yearn 3crypto with WETH", async () => {
```

Comment:
It seems to be completely identical to `Create with ETH and Deposit (uint256) in yearn 3crypto with WETH`. 
We need to fix this